index.html

```diff
@@ -180,30 +180,33 @@
       </div>
       <div class="row">
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Install Bazel</h3>
-          <p>See our <a href="docs/install.html">Installation Guide</a> to learn
-             how to download and install Bazel.
+          <h3>Java</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/java.html">Build Java tutorial</a>.
           </p>
         </div>
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Run Bazel</h3>
-          <p>See our
-             <a href="docs/getting-started.html">Getting Started Guide</a> to
-             learn how to write a BUILD file and run Bazel.
+          <h3>C++</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/cpp.html">Build C++ tutorial</a>.
           </p>
         </div>
       </div>
       <div class="row">
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Bazel Commands</h3>
-          <p>See our <a href="docs/bazel-user-manual.html">User Manual</a> to
-             learn about the Bazel command-line tool.
+          <h3>Android</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/app.html">mobile app tutorial</a>.
           </p>
         </div>
         <div class="col-sm-3 col-sm-offset-2">
-          <h3>Learn More</h3>
-          <p>Take a look at our <a href="docs/install.html">documentation</a>
-             and read our <a href="faq.html">FAQ</a>.
+          <h3>iOS</h3>
+          <p>Follow our <a href="{{ site.docs_site_url }}/tutorial/app.html">mobile app tutorial</a>.
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-4 col-sm-offset-4">
+          <h3>Other languages</h3>
+          <!-- Link to GitHub until we have a better {{ site.docs_site_url }}/be/overview.html-->
```

Comment:
@damienmg Are you referring to the sidebar menu or the landing page? Both the command line reference and the BE are in the sidebar, though the command line reference is located above the BE section, which may not be desirable since the BE is more useful than the command line reference.

@steren Thank you for the explanation. I certainly agree that the current BE (as well as the Skylark Library) has much room for improvement, and I look forward to going over the requested changes.

A few questions and comments on the issues you listed:

> the first part "Concepts and terminology" is really out of place and should not be here. "Bourne shell tokenization"? No, I am sorry, this is not how to start with Bazel.

Agreed that this list of links should be moved. This is an artifact of the way the internal BE has been for long time. The BE overview should focus on more on the rules.

> Objective C rules are deprecated in favor of rules_apple

Has this deprecation officially occurred yet or is it still in progress? If the native obj-c rules have been deprecated, then we also need to update the [iOS App Tutorial](https://bazel.build/versions/master/docs/tutorial/ios-app.html), which still recommends using the native rules.

> glob and filegroup are not supporting a getting started use case

They are not but `glob` are used quite commonly, especially with Java rules.

> the left nav does not auto-open and we have no complete list of rules appearing on that page.

Both of these issues can easily be fixed.

> we are maintaining many official rules that are not listed on this page, just only partially in the nav (TS, docker)...

Before we have a better way to publish and browse rules, we should add the missing links to the rules that we officially maintain since that is the current practice. We simply haven't made it a hard requirement when we open source new rule sets on bazelbuild.